front/src/modules/rollingStock/consts.ts

```diff
@@ -55,7 +60,7 @@ export const effortCurveCondKeys: EffortCurveCondKeys = {
 };
 
 export type RollingStockParametersValues = {
-  [key: string]: string | number | null | EffortCurves;
+  [key: string]: string | number | null | EffortCurves | RollingStockBase['power_restrictions'];
```

Comment:
Can you add this comment please ? For the refacto :)
```suggestion
  // TODO: remove this line in the type
  [key: string]: string | number | null | EffortCurves | RollingStockBase['power_restrictions'];
```

Reply:
Done